book/src/user/target-tier-policies.md

```diff
@@ -0,0 +1,288 @@
+including the devops team in the RFC proposing the target.
+
+- Tier 1 targets must have substantial, widespread interest within the
+  developer community, and must serve the ongoing needs of multiple production
+  users of Zebra across multiple organizations or projects. These requirements
+  are subjective, and determined by consensus of the approving teams. A tier 1
+  target may be demoted or removed if it becomes obsolete or no longer meets
+  this requirement.
+- The target maintainer team must include at least 2 developers.
+- The target must build and pass tests reliably in CI, for all components that
+  Zebra's CI considers mandatory.
+  - The target must not disable an excessive number of tests or pieces of tests
+    in the testsuite in order to do so. This is a subjective requirement.
+- Building the target and running the testsuite for the target must not take
+  substantially longer than other targets, and should not substantially raise
+  the maintenance burden of the CI infrastructure.
+  - In particular, if building the target takes a reasonable amount of time,
+    but the target cannot run the testsuite in a timely fashion due to low
+    performance of either native code or accurate emulation, that alone may
+    prevent the target from qualifying as tier 1.
+- If running the testsuite requires additional infrastructure (such as physical
+  systems running the target), the target maintainers must arrange to provide
+  such resources to the Zebra project, to the satisfaction and approval of the
+  Zebra devops team.
+  - Such resources may be provided via cloud systems, via emulation, or via
+    physical hardware.
+  - If the target requires the use of emulation to meet any of the tier
+    requirements, the approving teams for those requirements must have high
+    confidence in the accuracy of the emulation, such that discrepancies
+    between emulation and native operation that affect test results will
+    constitute a high-priority bug in either the emulation or the
+    implementation of the target.
+  - If it is not possible to run the target via emulation, these resources must
+    additionally be sufficient for the Zebra devops team to make them
+    available for access by Zebra team members, for the purposes of development
+    and testing. (Note that the responsibility for doing target-specific
+    development to keep the target well maintained remains with the target
+    maintainers. This requirement ensures that it is possible for other
+    Zebra developers to test the target, but does not obligate other Zebra
+    developers to make target-specific fixes.)
+  - Resources provided for CI and similar infrastructure must be available for
+    continuous exclusive use by the Zebra project. Resources provided
+    for access by Zebra team members for development and testing must be
+    available on an exclusive basis when in use, but need not be available on a
+    continuous basis when not in use.
```

Comment:
This doesn't really make sense for Zebra - we're not likely to use emulation or dedicated hardware.

Reply:
We've used emulation before (with ARM64) and might have to use it in the future:
https://github.com/ZcashFoundation/zebra/pull/4773#discussion_r922152609